### PR TITLE
🚸 Issue 641: add more NSFW domains to porn

### LINKED ISSUE #641: [Add request] Various NSFW imagebooru sites
URL you wish to be added:
rule34.paheal.net
konachan.com
yande.re

Why you believe this should be added: NSFW

Add to list: https://blocklistproject.github.io/Lists/porn.txt

Other info you think we should know:
konachan.net is a filtered version of the website

### PR DESCRIPTION
**Pull request by thomasmerz**
## Summary

This PR fixes #641 by adding some more NSFW domains to porn and it replaces PR #659 which was screwed up in some way 🤷🏻‍♂️ 

## Checklist

- [x] I have verified that I have not modified any files inside the `alt-version` folder (automated code will automatically update those files)

- [x] I have verified that I have not modified any files inside the `dnsmasq-version` folder (automated code will automatically update those files)

### CODE REVIEW

File: porn.txt [hunk 1]
```diff
@@ -1908842,9 +1908842,12 @@
...
+0.0.0.0 goldpornfilms.com
 0.0.0.0 bestandfree.com
 0.0.0.0 tubexclips.com
 0.0.0.0 lusthero.com
 0.0.0.0 teenda.com
-0.0.0.0 gfpimps.com
\ No newline at end of file
+0.0.0.0 gfpimps.com
+0.0.0.0 rule34.paheal.net
+0.0.0.0 konachan.com
```
Comment (spirillen) [inline porn.txt:1908852]: https://mypdns.org/my-privacy-dns/porn-records/-/issues/4422

File: porn.txt [hunk 2]
```diff
@@ -1908842,9 +1908842,12 @@
...
-0.0.0.0 goldpornfilms.com 
+0.0.0.0 goldpornfilms.com
 0.0.0.0 bestandfree.com
 0.0.0.0 tubexclips.com
 0.0.0.0 lusthero.com
 0.0.0.0 teenda.com
-0.0.0.0 gfpimps.com
\ No newline at end of file
+0.0.0.0 gfpimps.com
+0.0.0.0 rule34.paheal.net
```
Comment (spirillen) [inline porn.txt:1908851]: https://mypdns.org/my-privacy-dns/porn-records/-/issues/1345

File: porn.txt [hunk 3]
```diff
@@ -1908842,9 +1908842,12 @@
 0.0.0.0 xyzmilfs.com
 0.0.0.0 xyzteens.com
 0.0.0.0 xzxxtube.com
-0.0.0.0 goldpornfilms.com 
+0.0.0.0 goldpornfilms.com
```
Comment (spirillen) [inline porn.txt:1908845]: https://mypdns.org/my-privacy-dns/porn-records/-/issues/2455

File: porn.txt [hunk 4]
```diff
@@ -1908842,9 +1908842,12 @@
...
 0.0.0.0 bestandfree.com
 0.0.0.0 tubexclips.com
 0.0.0.0 lusthero.com
 0.0.0.0 teenda.com
-0.0.0.0 gfpimps.com
\ No newline at end of file
+0.0.0.0 gfpimps.com
```
Comment (spirillen) [inline porn.txt:1908850]: https://mypdns.org/my-privacy-dns/porn-records/-/issues/99893

File: porn.txt [hunk 5]
```diff
@@ -1908842,9 +1908842,12 @@
...
+0.0.0.0 rule34.paheal.net
+0.0.0.0 konachan.com
+0.0.0.0 yande.re
```
Comment (spirillen) [inline porn.txt:1908853]: https://mypdns.org/my-privacy-dns/porn-records/-/issues/94326

**Review by spirillen — CHANGES_REQUESTED** [review]
This is becoming to seems like a coincidence to much for ripping of from [Porn Records](https://mypdns.org/my-privacy-dns/porn-records) without proper crediting the use of our project as source to your blacklists

I will urge you to get the crediting in order and the information about the [LICENSE CC-BY-NC 4.0](https://mypdns.org/my-privacy-dns/porn-records/-/blob/master/LICENSE) 

This isn't my first comment on this subject.

This note will be recorded at mypdns.org for the future.

**Comment by thomasmerz** [comment]
@spirillen , do you mean me or @xjrb10 ?

File: porn.txt (hunk 2 above)
Comment (fishcharlie) [reply porn.txt:1908851]: @spirillen I think it's **EXTREMELY** hard to claim that this is a rip off of that issue. In that issue there are 7 examples of subdomains that it falls under. This adds 1 of those. If this was truly ripping from that source, the expectation would be to include all the domains.

However, just to verify, I have asked the original author of the issue where they found these domains at: https://github.com/blocklistproject/Lists/issues/641#issuecomment-1053724518

**Comment by cryptogap** [comment]
> This is becoming to seems like a coincidence to much for ripping of from [Porn Records](https://mypdns.org/my-privacy-dns/porn-records) without proper crediting the use of our project as source to your blacklists
> 
> I will urge you to get the crediting in order and the information about the [LICENSE CC-BY-NC 4.0](https://mypdns.org/my-privacy-dns/porn-records/-/blob/master/LICENSE)
> 
> This isn't my first comment on this subject.
> 
> This note will be recorded at mypdns.org for the future.

Just because a domain is on a list, any list DOES NOT mean that list in some way "owns" that domain. I would imagine that if I were to go through your lists there would be THOUSANDS of domain that are also on other lists. This does not mean that you ripped off other lists it merely means that both list are aware of the offending domain. 

Also, I have noticed that you have shilled your project on here multiple times including but not limited to speaking about your browser extension. Please do not shill your project here any more. Thank you

**Comment by spirillen** [comment]
@cryptogap 

Let's take this one from a new start, I can get hush too but for sure prefer cooperation between project, Right now I'm scanning your porn list with @pyfunceble by @funilrys.

From when I'm looking at the screen it actually looks like more than 50% of your list is simply dead records.

What I have doing by participating to the commits and issues on your board is a couple of thing, but the major reason have and would always had been sharing information with your project.

The other thing is I have kept the records license compliant.

> Just because a domain is on a list, any list DOES NOT mean that list in some way "owns" 

Well actually it does means that, it is the CC-by-NC 4. license, unlike your `The Unlicense` where you are giving up the rights to any of your code., db like lists are not necessary accepted as [software](https://github.com/blocklistproject/Lists/blob/6566933f6367e51bfbbc0fea8be4aa90dcf6505d/LICENSE#L1)...

They are more likely DB lists where the CC comes to play.

> Also, I have noticed that you have shilled your project on here multiple times including but not limited to speaking about your browser extension. Please do not shill your project here any more. Thank you

Sure, you solves the LICENSE and credit in the future by your self.

Stop harvest my and my users work from mypdns.org or your action might be followed up by a DMCA

-----------

My suggestion is instead of this threading tone of voice, we start collaborate with each other, there is enough war. mypdns PR is versus yours project geared and optimized for handling NSFW content, and as a open source project, you are actually free to copy from us, as long as there are no profit makers involved. (The NC in the license, Non Commercial)

--------

Let me know which trail you prefer.

**Comment by thomasmerz** [comment]
@spirillen , @fishcharlie , @cryptogap 
Do you have some news over here regarding cooperation (and working on this PR)?

BTW: @spirillen - why is [your gitlab down](https://github.com/thomasmerz/upptime/issues/995) since at least april, 10th? 😢

**Comment by spirillen** [comment]
> BTW: @spirillen - why is [your gitlab down](https://github.com/thomasmerz/upptime/issues/995) since at least april, 10th? cry

Well I have spend my last dime to try keeping the project alive, while people was using the work but not feeding us (€€€€), I  simply suffers from having any more electricity at home (for real). this means The server is now standing at a friends house, who have promised to host the server for a period of one year to see if it is possible to have people and companies to use the work and sponsor the project.

This means there will be a new reality about how the project will be listing new results...

1. A free open source list for not for profit as you know it, but only updated once or maybe twice a week
2. A commercial list which will be maintained real time as of today, but as said, this will change.

I sad it have come to this point where I have to put a price on it and I would hope people would just have sponsored enough to keep things running in a safe environment.

In the meantime I considering to change from gitlab software into youtrack + space as it seems to use way less resources.

If anybody is up for giving it a beta spin, it can be found at `http://support.mypdns.org:8080`

All comments are welcome, more feedback is always welcome

**Comment by thomasmerz** [comment]
@cryptogap resolved conflict. What about the discussions? 🤔

**Comment by spirillen** [comment]
> @cryptogap resolved conflict. What about the discussions? thinking

@thomasmerz if you did not harvest these form https://mypdns.org/my-privacy-dns/matrix then there are no issue, however if you did copy these domains from MyPDNS, then you need to add a notification about where they origin and they can not be used in commercial used list.

This means @thomasmerz, you have the answer

**Comment by thomasmerz** [comment]
@spirillen , are there still "ripping issues" or is this PR ok for you? If not, I will close my PR and this issue should be closed.

**Comment by thomasmerz** [comment]
You should better ask @xjrb10 who wanted these domains to be added - I just checked, found them add-worthy and made this PR… 🤷🏼‍♂️

**Comment by xjrb10** [comment]
@spirillen please refer to https://github.com/blocklistproject/Lists/issues/641#issuecomment-1053737459

These are already well known imagebooru sites, I already answered this ever since February which is literally 10 months ago.

**Comment by cryptogap** [comment]
@spirillen I am only going to mention this one more time. You do not own the right to have a domain on a list. The domains are public record. Please STOP harassing the members of this community. IF you continue to bring this up we will have no choice to ban you from this project AND report you to github for harassment. I really do NOT want to have to do this but if YOU continue YOU with leave us with no other options. 

Thank you and have a great day!

**Pull request merged**